requirements.txt

```diff
@@ -16,7 +16,7 @@ django-auth-ldap==1.2.6
 django-celery==3.1.17
 django-chunked-upload==1.1.0
 django-debug-toolbar==1.4
-django-docker-engine==0.0.42
+django-docker-engine==0.0.42.2
```

Comment:
@scottx611x : Could you file a bug against django_docker_engine for the error you hit earlier, that kept us from using the latest?